src/Orleans/Telemetry/Consumers/TraceTelemetryConsumer.cs

```diff
@@ -37,12 +36,21 @@ public void TrackTrace(string message, Severity severity)
                 case Severity.Off:
                     return;
             }
-            Trace.Flush();
```

Comment:
Ok, got it. So for Trace, each Trace.Write() is followed by a Trace.Flush()... Will check the old code and update the PR as soon as we have the xunit stuff done since the tests now are failing due to an old assembly attribute that was once used for MSTest. Thanks for the feedback.